AlexanderOtavka/ride-board
# Analytics

Start of work for #36. Puts together a data collection scaffold for tracking user behaviour, and sets up basic tracking for all page visits.

The big thing missing here is a way of viewing this data outside of pulling up a postgres shell in production, but that is being deferred for a future pr. 

# Manual Testing
1. `rake db:migrate`
2. Spin up and visit the page
3. Check the database for the new visit and event for the page visit. See Analytics.md for more info.
